saleor/graphql/account/mutations/staff/customer_update.py

```diff
@@ -162,3 +168,38 @@ def perform_mutation(cls, _root, info: ResolveInfo, /, **data):
 
         # Return the response
         return cls.success_response(new_instance)
+
+    @classmethod
+    @traced_atomic_transaction()
+    def save(cls, info: ResolveInfo, instance, cleaned_input):
+        default_shipping_address = cleaned_input.get(SHIPPING_ADDRESS_FIELD)
+        manager = get_plugin_manager_promise(info.context).get()
+
+        if default_shipping_address:
+            default_shipping_address.save()
+            instance.default_shipping_address = default_shipping_address
+
+        default_billing_address = cleaned_input.get(BILLING_ADDRESS_FIELD)
+
+        if default_billing_address:
+            default_billing_address.save()
+            instance.default_billing_address = default_billing_address
```

Comment:
Seems like address saving part is the same for all mutations, so maybe can be extracted to function and reused